BUILD.windows.md

```diff
@@ -3,43 +3,54 @@ Build instructions for Windows 10
 
 Environment Setup
 -----------------
+
 ### Chocolatey
+
 Install chocolatey, a package manager, to download the rest of the dependencies: <https://chocolatey.org/>
 
-Press Windows Key+X and Run Windows PowerShell(Admin) then follow the chocolatey instructions to "Install with Powershell.exe"
+Press Windows Key+X and Run Windows PowerShell(Admin) then follow the chocolatey instructions to "Install with
+Powershell.exe"
 
 After chocolatey is installed you can now install the rest of the dependencies:
 
     choco install visualstudio2019buildtools visualstudio2019-workload-vctools cmake ninja cmder qemu-img openssl -yfd
 
-You may have to disable Windows Defender Real-time protection if you want the packages to install quicker.
-Search for Windows Defender Security Center, go to Virus & threat protection, then Virus and thread protection settings, disable Real-time protection.
+You may have to disable Windows Defender Real-time protection if you want the packages to install quicker. Search for
+Windows Defender Security Center, go to Virus & threat protection, then Virus and thread protection settings, disable
+Real-time protection.
 
-NOTE: visualcpp-build-tools is only the installer package. For this reason, choco cannot detect any new compiler tool updates so choco upgrade
-will report no new updates available. To update the compiler and related tooling, you will need to search for "Add or remove programs",
-find "Microsoft Visual Studio Installer" and click "Modify".
+NOTE: visualcpp-build-tools is only the installer package. For this reason, choco cannot detect any new compiler tool
+updates so choco upgrade will report no new updates available. To update the compiler and related tooling, you will need
+to search for "Add or remove programs", find "Microsoft Visual Studio Installer" and click "Modify".
 
 ### Git
-You need to enable symlinks in Windows Git, have a look at [the git-for-windows docs](https://github.com/git-for-windows/git/wiki/Symbolic-Links).
+
+You need to enable symlinks in Windows Git, have a look at
+[the git-for-windows docs](https://github.com/git-for-windows/git/wiki/Symbolic-Links).
 
 ### Qt6
+
 Install the latest stable version of Qt6 (6.2.4 at the moment): <https://www.qt.io/download-thank-you?os=windows/>.
```

Comment:
`6.2.4` is no longer the "latest" stable version.

Reply:
IMO, we can just get rid of the version information, unless we want to pin QT to a specific stable version.

Reply:
Either that or "latest stable version". But this is kind of outside scope here, I just want to have all platforms linked from the Readme... I'll open an issue for improvements specifically.